homeassistant/components/rfxtrx.py

```diff
@@ -173,13 +216,22 @@ def handle_receive(event):
     device = config[DOMAIN][ATTR_DEVICE]
     debug = config[DOMAIN][ATTR_DEBUG]
     dummy_connection = config[DOMAIN][ATTR_DUMMY]
+    recv_modes = config[DOMAIN][ATTR_RECV_MODES]
+
+    if recv_modes is not None:
+        for mode in recv_modes:
+            if rfxtrxmod.lowlevel.get_recmode_tuple(mode) == (None, None):
+                _LOGGER.error('recv_mode "%s" is unknown to RFXTRX.', mode)
+                return False
+        _LOGGER.debug("Receiving modes to be enabled: %s", recv_modes)
```

Comment:
I think you can remove line 222-226, since it is included in the valid_recvmode()